types/dsinjs__binary-tree/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "lib": [
+      "ES2018",
+      "ES2019",
+      "es2017",
+      "es7",
+      "es6",
+      "es5",
+      "dom"
+    ],
+    "strict": true,
```

Comment:
I think we've been trying to converge on this convention
```json
        "paths":{
            "@dsinjs/*": ["dsinjs__*"]
        },
```

Reply:
Sure I will follow these rules,
Sorry was not aware of this.

Reply:
Not your fault - it's new and undocumented.  Thanks for updating!